ses-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,6 @@
+[project]
+org-name = "ballerinax"
+version = "0.6.0"
```

Comment:
If this is for swarn lake release, better we change this version to `0.9.0`

Reply:
Changed the version.